bc_obps/reporting/migrations/0141_populate_report_open_date.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 5.1.14 on 2025-11-18 23:30
+
+from datetime import datetime
+from django.db import migrations
+from django.utils import timezone
+
+
+def populate_report_open_date(apps, schema_editor):
+    """
+    Populate report_open_date for existing reporting years.
+    Sets it to March 1st at midnight UTC for the year after the reporting year.
+    """
+    ReportingYear = apps.get_model('reporting', 'ReportingYear')
+
+    reporting_years = ReportingYear.objects.all()
+    for reporting_year in reporting_years:
+        if not reporting_year.report_open_date:
+            # Set to March 1st at midnight UTC for the year after the reporting year
+            next_year = reporting_year.reporting_year + 1
+            reporting_year.report_open_date = timezone.make_aware(datetime(next_year, 3, 1, 0, 0, 0))
+
+    # Bulk update all reporting years
+    ReportingYear.objects.bulk_update(reporting_years, ['report_open_date'])
+
+
+def reverse_populate_report_open_date(apps, schema_editor):
+    """
+    Reverse migration - set report_open_date to None for all reporting years.
+    """
+    ReportingYear = apps.get_model('reporting', 'ReportingYear')
+    ReportingYear.objects.all().update(report_open_date=None)
```

Comment:
This is actually not true. I'd make this migration forward-only, or just not do anything on reverse. We have no way of knowing the state of the `report_open_date` field prior to this migration